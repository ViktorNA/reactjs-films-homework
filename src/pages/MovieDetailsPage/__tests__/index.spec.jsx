import TestRenderer from 'react-test-renderer';
import React from 'react';
import Page from '../index';

describe('Snapshot test', () => {
  it('works correctly', () => {
    const testRender = TestRenderer.create(<Page />);
    expect(testRender.toJSON).toMatchSnapshot();
  });
});
