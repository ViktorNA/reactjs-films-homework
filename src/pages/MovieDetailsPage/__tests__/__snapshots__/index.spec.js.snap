// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test works correctly 1`] = `[Function]`;
